osbs/repo_utils.py

```diff
@@ -60,7 +60,9 @@ def __init__(self, dir_path='', file_name=REPO_CONFIG_FILE):
             with open(file_path) as f:
                 self.container = (yaml.load(f) or {})
 
-        modules = self.container.get('compose', {}).get('modules', [])
+        # container values may be set to None
+        container_compose = self.container.get('compose', {}) or {}
```

Comment:
IMO we can use only get() without default value, to avoid duplicated definitions of default.